packages/quantic/force-app/examples/main/lwc/exampleQuanticFoldedResultList/resultTemplates/childTemplate.html

```diff
@@ -0,0 +1,20 @@
+<template>
+  <c-quantic-result-template is-any-preview-open={isAnyPreviewOpen} hide-bottom-border
```

Comment:
The `hide-bottom-border` attribute should not be set anymore.

```suggestion
  <c-quantic-result-template is-any-preview-open={isAnyPreviewOpen}
```